### PR TITLE
added filesize show in kB/MB/GB etc

### PR DESCRIPTION
**Pull request by MicSm**
version 1.1.2

### CODE REVIEW

File: src/extension.js [hunk 1]
```diff
@@ -322,6 +322,19 @@ async function do_gunzip(input, output) {
 	await pipe(source, gunzip, destination);
 }
 
+function nBytes(x){
+	
+	const units = ['bytes', 'kB', 'MB', 'GB', 'TB', 'PB', 'EB', 'ZB', 'YB'];
```
Comment (Kandimus) [inline src/extension.js:327]: GB and highter. Lol

File: src/extension.js [hunk 2]
```diff
@@ -322,6 +322,19 @@ async function do_gunzip(input, output) {
...
+	let l = 0, n = parseInt(x, 10) || 0;
+	
+	while (n >= 1000) {
```
Comment (Kandimus) [inline src/extension.js:330]: may be 1024? upss....

File: src/extension.js [hunk 3]
```diff
@@ -470,7 +483,7 @@ async function loadCTags(tagFilePath)
 		var liner = new LineByLine(tagFilePath);
 	}
 	catch(err) {
-		vscode.window.showErrorMessage("Can not load file '.ctags'");
+		vscode.window.showErrorMessage("Can't load CTags file");
```
Comment (Kandimus) [inline src/extension.js:486]: Maybe then change it to something like "Can't load file ${tagFilePath} CTags!"?

File: src/extension.js (hunk 2 above)
Comment (MicSm) [reply src/extension.js:330]: no ;-) fixed

File: src/extension.js (hunk 1 above)
Comment (MicSm) [reply src/extension.js:327]: fixed

File: src/extension.js (hunk 3 above)
Comment (MicSm) [reply src/extension.js]: fixed

**Review by Kandimus — APPROVED** [review]

**Pull request merged**